dallinger/config.py

```diff
@@ -53,7 +59,8 @@
     ("num_dynos_worker", int, []),
     ("organization_name", six.text_type, []),
     ("port", int, ["PORT"]),
-    ("qualification_blacklist", six.text_type, []),
+    ("mturk_qualification_blocklist", six.text_type, ["qualification_blacklist"]),
```

Comment:
Are you sure this works? I think I tried this name switch a couple years ago and found that config.txt files with the old key did not get read the way I expected.

Reply:
Confirmed working by @MatthewWilkes

Reply:
It seems to. If you use the old name in tests they still pass, and setting a value using the old name then doing debug shows the value associated with the new name TTW.